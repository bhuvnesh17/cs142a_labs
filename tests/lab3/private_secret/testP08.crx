// deeply nested else's

func main() : void {
  var a1 : int;
  
  if (true) {
    ::printInt(a1);
    ::printInt(a2);
    ::printInt(a3);
    ::printInt(a4);
    ::printInt(a5);
    ::printInt(a6);
    ::printInt(a7);
    ::printInt(a8);
    ::printInt(a9);
    ::printInt(a10);
  } else {
    var a2 : int;
    
    if (true) {
      ::printInt(a1);
      ::printInt(a2);
      ::printInt(a3);
      ::printInt(a4);
      ::printInt(a5);
      ::printInt(a6);
      ::printInt(a7);
      ::printInt(a8);
      ::printInt(a9);
      ::printInt(a10);
    } else {
      var a3 : int;

      if (true) {
        ::printInt(a1);
        ::printInt(a2);
        ::printInt(a3);
        ::printInt(a4);
        ::printInt(a5);
        ::printInt(a6);
        ::printInt(a7);
        ::printInt(a8);
        ::printInt(a9);
        ::printInt(a10);
      } else {
        var a4 : int;

        if (true) {
          ::printInt(a1);
          ::printInt(a2);
          ::printInt(a3);
          ::printInt(a4);
          ::printInt(a5);
          ::printInt(a6);
          ::printInt(a7);
          ::printInt(a8);
          ::printInt(a9);
          ::printInt(a10);
        } else {
          var a5 : int;

          if (true) {
            ::printInt(a1);
            ::printInt(a2);
            ::printInt(a3);
            ::printInt(a4);
            ::printInt(a5);
            ::printInt(a6);
            ::printInt(a7);
            ::printInt(a8);
            ::printInt(a9);
            ::printInt(a10);
          } else {
            var a6 : int;

            if (true) {
              ::printInt(a1);
              ::printInt(a2);
              ::printInt(a3);
              ::printInt(a4);
              ::printInt(a5);
              ::printInt(a6);
              ::printInt(a7);
              ::printInt(a8);
              ::printInt(a9);
              ::printInt(a10);
            } else {
              var a7 : int;

              if (true) {
                ::printInt(a1);
                ::printInt(a2);
                ::printInt(a3);
                ::printInt(a4);
                ::printInt(a5);
                ::printInt(a6);
                ::printInt(a7);
                ::printInt(a8);
                ::printInt(a9);
                ::printInt(a10);
              } else {
                var a8 : int;

                if (true) {
                  ::printInt(a1);
                  ::printInt(a2);
                  ::printInt(a3);
                  ::printInt(a4);
                  ::printInt(a5);
                  ::printInt(a6);
                  ::printInt(a7);
                  ::printInt(a8);
                  ::printInt(a9);
                  ::printInt(a10);
                } else {
                  var a9 : int;

                  if (true) {
                    ::printInt(a1);
                    ::printInt(a2);
                    ::printInt(a3);
                    ::printInt(a4);
                    ::printInt(a5);
                    ::printInt(a6);
                    ::printInt(a7);
                    ::printInt(a8);
                    ::printInt(a9);
                    ::printInt(a10);
                  } else {
                    var a10 : int;

                    ::printInt(a1);
                    ::printInt(a2);
                    ::printInt(a3);
                    ::printInt(a4);
                    ::printInt(a5);
                    ::printInt(a6);
                    ::printInt(a7);
                    ::printInt(a8);
                    ::printInt(a9);
                    ::printInt(a10);
                  }
                  ::printInt(a1);
                  ::printInt(a2);
                  ::printInt(a3);
                  ::printInt(a4);
                  ::printInt(a5);
                  ::printInt(a6);
                  ::printInt(a7);
                  ::printInt(a8);
                  ::printInt(a9);
                  ::printInt(a10);
                }
                ::printInt(a1);
                ::printInt(a2);
                ::printInt(a3);
                ::printInt(a4);
                ::printInt(a5);
                ::printInt(a6);
                ::printInt(a7);
                ::printInt(a8);
                ::printInt(a9);
                ::printInt(a10);
              }
              ::printInt(a1);
              ::printInt(a2);
              ::printInt(a3);
              ::printInt(a4);
              ::printInt(a5);
              ::printInt(a6);
              ::printInt(a7);
              ::printInt(a8);
              ::printInt(a9);
              ::printInt(a10);
            }
            ::printInt(a1);
            ::printInt(a2);
            ::printInt(a3);
            ::printInt(a4);
            ::printInt(a5);
            ::printInt(a6);
            ::printInt(a7);
            ::printInt(a8);
            ::printInt(a9);
            ::printInt(a10);
          }
          ::printInt(a1);
          ::printInt(a2);
          ::printInt(a3);
          ::printInt(a4);
          ::printInt(a5);
          ::printInt(a6);
          ::printInt(a7);
          ::printInt(a8);
          ::printInt(a9);
          ::printInt(a10);
        }
        ::printInt(a1);
        ::printInt(a2);
        ::printInt(a3);
        ::printInt(a4);
        ::printInt(a5);
        ::printInt(a6);
        ::printInt(a7);
        ::printInt(a8);
        ::printInt(a9);
        ::printInt(a10);
      }
      ::printInt(a1);
      ::printInt(a2);
      ::printInt(a3);
      ::printInt(a4);
      ::printInt(a5);
      ::printInt(a6);
      ::printInt(a7);
      ::printInt(a8);
      ::printInt(a9);
      ::printInt(a10);
    }
    ::printInt(a1);
    ::printInt(a2);
    ::printInt(a3);
    ::printInt(a4);
    ::printInt(a5);
    ::printInt(a6);
    ::printInt(a7);
    ::printInt(a8);
    ::printInt(a9);
    ::printInt(a10);
  }
  ::printInt(a1);
  ::printInt(a2);
  ::printInt(a3);
  ::printInt(a4);
  ::printInt(a5);
  ::printInt(a6);
  ::printInt(a7);
  ::printInt(a8);
  ::printInt(a9);
  ::printInt(a10);
}
