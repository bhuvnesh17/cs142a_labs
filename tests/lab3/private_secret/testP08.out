Error parsing file.
ResolveSymbolError(8,16)[Could not find a2.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)

ResolveSymbolError(9,16)[Could not find a3.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)

ResolveSymbolError(10,16)[Could not find a4.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)

ResolveSymbolError(11,16)[Could not find a5.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)

ResolveSymbolError(12,16)[Could not find a6.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)

ResolveSymbolError(13,16)[Could not find a7.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)

ResolveSymbolError(14,16)[Could not find a8.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)

ResolveSymbolError(15,16)[Could not find a9.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)

ResolveSymbolError(16,16)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)

ResolveSymbolError(23,18)[Could not find a3.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)

ResolveSymbolError(24,18)[Could not find a4.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)

ResolveSymbolError(25,18)[Could not find a5.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)

ResolveSymbolError(26,18)[Could not find a6.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)

ResolveSymbolError(27,18)[Could not find a7.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)

ResolveSymbolError(28,18)[Could not find a8.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)

ResolveSymbolError(29,18)[Could not find a9.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)

ResolveSymbolError(30,18)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)

ResolveSymbolError(38,20)[Could not find a4.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)

ResolveSymbolError(39,20)[Could not find a5.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)

ResolveSymbolError(40,20)[Could not find a6.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)

ResolveSymbolError(41,20)[Could not find a7.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)

ResolveSymbolError(42,20)[Could not find a8.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)

ResolveSymbolError(43,20)[Could not find a9.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)

ResolveSymbolError(44,20)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)

ResolveSymbolError(53,22)[Could not find a5.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)

ResolveSymbolError(54,22)[Could not find a6.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)

ResolveSymbolError(55,22)[Could not find a7.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)

ResolveSymbolError(56,22)[Could not find a8.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)

ResolveSymbolError(57,22)[Could not find a9.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)

ResolveSymbolError(58,22)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)

ResolveSymbolError(68,24)[Could not find a6.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)

ResolveSymbolError(69,24)[Could not find a7.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)

ResolveSymbolError(70,24)[Could not find a8.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)

ResolveSymbolError(71,24)[Could not find a9.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)

ResolveSymbolError(72,24)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)

ResolveSymbolError(83,26)[Could not find a7.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)

ResolveSymbolError(84,26)[Could not find a8.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)

ResolveSymbolError(85,26)[Could not find a9.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)

ResolveSymbolError(86,26)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)

ResolveSymbolError(98,28)[Could not find a8.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)
              Symbol(a7)

ResolveSymbolError(99,28)[Could not find a9.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)
              Symbol(a7)

ResolveSymbolError(100,28)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)
              Symbol(a7)

ResolveSymbolError(113,30)[Could not find a9.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)
              Symbol(a7)
                Symbol(a8)

ResolveSymbolError(114,30)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)
              Symbol(a7)
                Symbol(a8)

ResolveSymbolError(128,32)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)
              Symbol(a7)
                Symbol(a8)
                  Symbol(a9)

ResolveSymbolError(152,30)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)
              Symbol(a7)
                Symbol(a8)
                  Symbol(a9)

ResolveSymbolError(162,28)[Could not find a9.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)
              Symbol(a7)
                Symbol(a8)

ResolveSymbolError(163,28)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)
              Symbol(a7)
                Symbol(a8)

ResolveSymbolError(172,26)[Could not find a8.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)
              Symbol(a7)

ResolveSymbolError(173,26)[Could not find a9.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)
              Symbol(a7)

ResolveSymbolError(174,26)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)
              Symbol(a7)

ResolveSymbolError(182,24)[Could not find a7.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)

ResolveSymbolError(183,24)[Could not find a8.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)

ResolveSymbolError(184,24)[Could not find a9.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)

ResolveSymbolError(185,24)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)
            Symbol(a6)

ResolveSymbolError(192,22)[Could not find a6.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)

ResolveSymbolError(193,22)[Could not find a7.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)

ResolveSymbolError(194,22)[Could not find a8.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)

ResolveSymbolError(195,22)[Could not find a9.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)

ResolveSymbolError(196,22)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)
          Symbol(a5)

ResolveSymbolError(202,20)[Could not find a5.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)

ResolveSymbolError(203,20)[Could not find a6.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)

ResolveSymbolError(204,20)[Could not find a7.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)

ResolveSymbolError(205,20)[Could not find a8.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)

ResolveSymbolError(206,20)[Could not find a9.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)

ResolveSymbolError(207,20)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)
        Symbol(a4)

ResolveSymbolError(212,18)[Could not find a4.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)

ResolveSymbolError(213,18)[Could not find a5.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)

ResolveSymbolError(214,18)[Could not find a6.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)

ResolveSymbolError(215,18)[Could not find a7.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)

ResolveSymbolError(216,18)[Could not find a8.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)

ResolveSymbolError(217,18)[Could not find a9.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)

ResolveSymbolError(218,18)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)
      Symbol(a3)

ResolveSymbolError(222,16)[Could not find a3.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)

ResolveSymbolError(223,16)[Could not find a4.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)

ResolveSymbolError(224,16)[Could not find a5.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)

ResolveSymbolError(225,16)[Could not find a6.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)

ResolveSymbolError(226,16)[Could not find a7.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)

ResolveSymbolError(227,16)[Could not find a8.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)

ResolveSymbolError(228,16)[Could not find a9.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)

ResolveSymbolError(229,16)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)
    Symbol(a2)

ResolveSymbolError(232,14)[Could not find a2.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)

ResolveSymbolError(233,14)[Could not find a3.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)

ResolveSymbolError(234,14)[Could not find a4.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)

ResolveSymbolError(235,14)[Could not find a5.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)

ResolveSymbolError(236,14)[Could not find a6.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)

ResolveSymbolError(237,14)[Could not find a7.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)

ResolveSymbolError(238,14)[Could not find a8.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)

ResolveSymbolError(239,14)[Could not find a9.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)

ResolveSymbolError(240,14)[Could not find a10.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a1)


